source/_components/light.mysensors.markdown

```diff
@@ -108,6 +108,138 @@ void incomingMessage(const MyMessage &message) {
     }
 }
 ```
+### {% linkable_title MySensors 2.x example sketch %}
 
+```cpp
+/* 
+ * Example Dimmable Light
+ * Code adapted from http://github.com/mysensors/MySensors/tree/master/examples/DimmableLight
+ * 
+ * Documentation: http://www.mysensors.org
+ * Support Forum: http://forum.mysensors.org
+ *
+ */
+
+// Enable debug prints
+#define MY_DEBUG
+
+// Enable and select radio type attached
+#define MY_RADIO_NRF24
+//#define MY_RADIO_RFM69
+
+#include <MySensors.h>
+
+#define CHILD_ID_LIGHT 1
+
+#define LIGHT_OFF 0
+#define LIGHT_ON 1
+
+#define SN "Dimmable Light"
+#define SV "1.0"
+
+int16_t last_state = LIGHT_ON;
+int16_t last_dim = 100;
+
+MyMessage light_msg( CHILD_ID_LIGHT, V_LIGHT );
+MyMessage dimmer_msg( CHILD_ID_LIGHT, V_DIMMER );
```

Comment:
V_LIGHT and V_DIMMER are deprecated in mysensors 2.0. The new values are called V_STATUS and V_PERCENTAGE. The deprecated values still work, but I think we should use the new values in the sketch.